base_layer/wallet/src/wallet.rs

```diff
@@ -313,4 +313,38 @@ where
             .block_on(self.output_manager_service.sync_with_base_node())?;
         Ok(request_key)
     }
+
+    /// Do a coin split
+    pub fn coin_spit(
```

Comment:
expose the message as an argument too please.

Reply:
Done